src/components/EditView.js

```diff
@@ -49,17 +50,39 @@ const EditView = (props) => {
   }
   return (
     <form onSubmit={handleSubmit}>
-      <label htmlFor='task' />
-      <input placeholder={task} value={todo} onChange={handleChange} name='task' />
-      <label htmlFor='completed'>Task Completed: </label>
-      <input
-        type='checkbox'
-        name='completed'
-        onChange={handleChecked}
-        value={done}
-        checked={completed}
-      />
-      <button>Save</button>
+      <Flex p='10px' direction='column' align='center' justify='center'>
+        <label htmlFor='task'>Task: </label>
+        <Input
+          placeholder={task}
+          value={todo}
+          onChange={handleChange}
+          name='task'
+          variant='outline'
+          size='md'
+          _placeholder={{ opacity: 0.4, color: 'blue.300' }}
+        />
+        <label htmlFor='completed'>Task Completed: </label>
+        <input
```

Comment:
Should this be `Input` component?

Reply:
Chakra has a Checkbox component or a useCheckbox hook.  I prefer for now just to use the plain checkbox.